source/perflib/basename.c

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2024 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <string.h>
+
+#include <openssl/crypto.h>
+
+/*
+ * windows variant of basename(3). works on ASCIIZ only.
```

Comment:
Is this a Windows-only variant, or is it a Windows-compatible variant?

Reply:
this is implementation of basename(3) for windows. I could not find a basename() equivalent on windows. The basename is defined by POSIX.1 (I think).